widget/embedded/src/store/slices/config.ts

```diff
@@ -1,8 +1,11 @@
 import type { WidgetConfig } from '../../types';
 import type { StateCreatorWithInitialData } from '../app';
 
+import { i18n } from '@lingui/core';
+
 export const DEFAULT_CONFIG: WidgetConfig = {
   apiKey: '',
+  title: i18n.t('Swap'),
```

Comment:
If it has default value, why the type is optional and in `widget/embedded/src/pages/Home.tsx` we've added a fallback? 
Please use `undefined` here or if you want to keep this value here, update the type to be only string and remove the condition you've added to `widget/embedded/src/pages/Home.tsx`

Reply:
I changed it to `""`

Reply:
So why the type for `title` has `undefined`? When it happens?

Reply:
Because it is possible for the user to not send any title as props and if I remove the undefined, the user will encounter a typescript error indicating that the title can not be undefined.

Reply:
Got it, so using `undefiend` for initial value is more appropriate.

Reply:
Changed